frontend/src/app/modules/team/team-info/team-projects/team-projects.component.html

```diff
@@ -14,6 +14,6 @@
             <button icon="pi pi-minus-circle" class="p-button-outlined p-button-danger" pButton label="Remove"
                 (click)="removeFromTeam(projectTeam.id)"></button>
         </div>
-        <div class="panel-list-item" *ngIf="projectTeams?.length===0">No projects yet...</div>
+        <div class="panel-list-item placeholder-empty" *ngIf="projectTeams?.length===0">No projects yet...</div>
```

Comment:
Why didn't you just use !projectTeams?.length

Reply:
Fixed